redhat-ipaas-dev-single-tenant.yml

```diff
@@ -95,6 +95,10 @@ parameters:
   name: POSTGRESQL_VOLUME_CAPACITY
   required: true
   value: 1Gi
+- description: Ignore SSL errors in proxy containers
+  displayName: Insecure skip verify
+  name: INSECURE_SKIP_VERIFY
+  value: 'true'
```

Comment:
What is the default of 'token-rp' ? I would love to stick to this default here, too (probably 'false') and switch it on if required.